### PR TITLE
WIP: Drop bootstrap.js

### PR DESCRIPTION
**Pull request by staabm**
motivation: the big main.min.js is the biggest offender as far as client side performance is a concern.
In case we can merge this it should be way easier to get rid of the jquery dependency which contains a lot of unnecessary javascript right now.

The bootstrap.js is a big junk of javascript and it seems we dont explicitly use it anywhere.

Would be great to get this PR on a platform.sh test-deployment so I can verify nothing broke with this commit

### CODE REVIEW

**Comment by staabm** [comment]
Initla test on ipad looks good. Need another round of testing on desktop firefox/chrome.

Also need to verify the „next unread message“ feature works. It seems that it might got afffected by the change when testing on ipad - cannot say for sure because no browser console available

**Comment by mnapoli** [comment]
I had a look and there's this line using Bootstrap's JS:

https://github.com/mnapoli/externals/blob/cf0550c6d6ec58f28e6041630a0ae5f4859a296c/res/assets/main.js#L2

But it's not working on master anyway. I guess we can remove it as well.

**Comment by mnapoli** [comment]
Oh wait there is the feature to collapse threads that relies on this:

https://github.com/mnapoli/externals/blob/6fc2bcf56317ddaa133c7197a229d8ecd7b7f6a3/res/views/thread.html.twig#L13

**Comment by staabm** [comment]
Thx for the hints. I will try to work on gettings this features in in vanillajs

File: res/views/thread.html.twig [hunk 1]
```diff
@@ -10,7 +10,7 @@
     <article id="{{ email.number }}" class="clearfix {% if not email.isRead %}unread{% endif %}">
 
         <header>
-            <a href="#{{ email.number }}-body" role="button" data-toggle="collapse"><span class="fa fa-compress"></span></a>
+            <a data-target="#{{ email.number }}-body" role="button" data-toggle="collapse"><span class="fa fa-compress"></span></a>
```
Comment (staabm) [inline res/views/thread.html.twig]: No href means link is not properly accessible on ipad

**Comment by gharlan** [comment]
> I had a look and there's this line using Bootstrap's JS:
> $('[data-toggle="tooltip"]').tooltip(); 
> But it's not working on master anyway. I guess we can remove it as well.

There is one usage of it, and for me it is working on master:
screenshot 2019-02-16 13 50 40

https://github.com/mnapoli/externals/blob/52d33560131fd333676dfa52767033e9d7f50d36/res/views/layout.html.twig#L60

**Comment by gharlan** [comment]
But maybe the native title tooltip is enough here?

**Comment by staabm** [comment]
Even a small self contained library for the tooltip would fine IMO (in case we are not ok with the native title attributes) as long as we can get rid of the bootstrap.js

**Pull request closed without merging**